tests/testthat/helper_testthat_correctness_transactions.R

```diff
@@ -231,3 +231,37 @@ fct.testthat.correctness.clvfittedtransactions <- function(name.model, method, d
   fct.testthat.correctness.clvfittedtransactions.staticcov.regularization.lambda.0.no.regularization(method = method, clv.apparel.staticcov = clv.apparel.staticcov,
                                                                                                      m.fitted.static = obj.fitted.static)
 }
+
+fct.testthat.correctness.clvfittedtransactions.common <- function(fct.expectation.R, fct.expectation.Cpp, params_i, obj.fitted){
+  dt.expectation.seq <- clv.time.expectation.periods(clv.time = obj.fitted@clv.data@clv.time,
```

Comment:
name method more descriptive. Something like`fct.testthat.correctness.clvfittedtransactions.same.expectation.in.R.and.Cpp`